src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.OpenFileTracker.cs

```diff
@@ -24,36 +22,8 @@ internal partial class VisualStudioWorkspaceImpl
         /// <summary>
         /// Singleton the subscribes to the running document table and connects/disconnects files to files that are opened.
         /// </summary>
-        public sealed class OpenFileTracker
+        public sealed class VisualStudioOpenFileTracker : AbstractOpenFileTracker<VisualStudioWorkspaceImpl>
```

Comment:
```suggestion
        public sealed class VisualStudioWorkspaceOpenFileTracker : AbstractOpenFileTracker<VisualStudioWorkspaceImpl>
```

The reason being that sometimes we use "VisualStudioBlah" to generally mean an implementation that's used for all workspace implementations running in VS; this isn't quite the same.